client/src/main/java/org/apache/rocketmq/client/impl/consumer/RebalanceImpl.java

```diff
@@ -521,7 +521,13 @@ private boolean updateProcessQueueTableInRebalance(final String topic, final Set
                 this.removeDirtyOffset(mq);
                 ProcessQueue pq = createProcessQueue(topic);
                 pq.setLocked(true);
-                long nextOffset = this.computePullFromWhere(mq);
+                long nextOffset;
+                try {
+                    nextOffset = this.computePullFromWhereWithException(mq);
+                } catch (MQClientException e) {
+                    log.info("doRebalance, {}, compute offset failed, {}", consumerGroup, mq);
```

Comment:
maybe log.info() can be replaced with log.warn()?
Does it need to be consistent with the following codes?
` log.warn("doRebalance, {}, add new mq failed, {}", consumerGroup, mq);`

Reply:
thanks for your comment. sure, it should be log.warn(), i will change it later